### PR TITLE
update ocp version and Grafana dashboards

### PR DESCRIPTION
**Pull request by ebattat**
1. Update ocp version from upstream URL.
2. Update latest grafana dashboards

### CODE REVIEW

**Comment by ebattat** [comment]
We are fetching the OCP version from the cluster before having the new cluster credentials, so solve it by fetching it from assisted installer upstream url.

**Comment by RobertKrawitz** [comment]
Looking at `OCPVersions.get_latest_version`, it looks like this simply gets the latest available version, not necessarily the one we're installing.

**Comment by ebattat** [comment]
@RobertKrawitz, it preferred way to show which version installed instead of not showing at all.

**Comment by RobertKrawitz** [comment]
What happens if we deliberately install a version other than the very latest version available (e. g. we choose to install 4.11 even after 4.12 is released)?

**Comment by ebattat** [comment]
@RobertKrawitz, you right fix it. in case of passing an constant version, I will take it as it, meaning the version you pass from [INSTALL_OCP_VERSION environment variable](https://github.com/redhat-performance/benchmark-runner/blob/ea82ed22997cc0582653e4493a242857234760b3/.github/workflows/Weekly_Perf_Env_Installer_CI.yml#L67).

**Comment by ebattat** [comment]
@RobertKrawitz agreed, fixed.

File: benchmark_runner/common/clouds/IBM/ibm_operations.py [hunk 1]
```diff
@@ -209,7 +208,7 @@ def version_already_installed(self):
         """
         if self.get_ocp_server_version() == 'null':
             return False
-        elif self.LATEST_VERSION in self.__install_ocp_version and self.get_ocp_server_version() == self.__get_latest_version():
+        elif self.LATEST_VERSION in self.__install_ocp_version and self.get_ocp_server_version() == self.__get_installation_version():
```
Comment (RobertKrawitz) [inline benchmark_runner/common/clouds/IBM/ibm_operations.py:211]: LATEST_VERSION appears to be the fixed string `latest`.  I'm not quite sure how that's working here?

Comment (ebattat) [reply benchmark_runner/common/clouds/IBM/ibm_operations.py:211]: What do you mean, LATEST instead of LATEST_VERSION ?

Comment (RobertKrawitz) [reply benchmark_runner/common/clouds/IBM/ibm_operations.py]: That wasn't what I meant, and no need to change it.  I'm trying to work out what happens if the version specified is `latest`, how that gets translated into a specific version.

Comment (ebattat) [reply benchmark_runner/common/clouds/IBM/ibm_operations.py]: This is already working. Please let me know if you have more comments.

Comment (RobertKrawitz) [reply benchmark_runner/common/clouds/IBM/ibm_operations.py]: I'll approve it as I don't believe you have made any material change, but I would like to understand how specifying `latest` works.

Comment (RobertKrawitz) [reply benchmark_runner/common/clouds/IBM/ibm_operations.py]: Before I approve it -- I notice that the current diff has a lot of changes to various JSON files.  What's that about?

Comment (ebattat) [reply benchmark_runner/common/clouds/IBM/ibm_operations.py]: I have update our grafana dashboards, adding it into description.

Comment (RobertKrawitz) [reply benchmark_runner/common/clouds/IBM/ibm_operations.py]: That ideally should be a separate PR, but I'll OK this.  I've updated the title too.

**Review by RobertKrawitz — APPROVED** [review]

**Pull request merged**